tests/tests_pytorch/loggers/conftest.py

```diff
@@ -141,6 +141,10 @@ def __setitem__(self, key, value):
     neptune_utils.stringify_unsupported = Mock()
     monkeypatch.setitem(sys.modules, "neptune.utils", neptune_utils)
 
+    neptune_exceptions = ModuleType("exceptions")
+    neptune_exceptions.InactiveRunException = Exception
```

Comment:
what does this line do?

Reply:
`neptune` may not be installed when running tests so we're using monkey patching to patch imports with some stubs.